docs/user-guide/permissions/index.md

```diff
@@ -0,0 +1,8 @@
+Atomist integrates with and automates activities over various tools and services in your development toolchain. To do so, certain permission are needed. We aim to ask only for the permissions required to get the job done.
+
+Please see the individual sections for specifics on the permissions requested by Atomist.
+
+* [Slack App OAuth Scopes](slack.md)
+* [GitHub OAuth Scopes](github.md)
```

Comment:
Technically the text for list items is supposed to start in column 4.  MkDocs and GitHub are forgiving on this, some Markdown renderers are not. Keeping with the standard allows more easy migration away from MkDocs should we ever need to.